### PR TITLE
dnsviz 0.9.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> /opt/homebrew/Cellar/dnsviz/0.9.4/bin/dnsviz probe -d 0 -A -x example.com:example.com.zone.signed -N example.com:example.com.zone-delegation -D example.com:example.com.zone-delegation -o example.com.json example.com
Error: test failed
Error: dnspython does not appear to be installed
```

File: Formula/dnsviz.rb [hunk 1]
```diff
@@ -26,21 +26,6 @@ class Dnsviz < Formula
...
-  resource "M2Crypto" do
-    url "https://files.pythonhosted.org/packages/aa/36/9fef97358e378c1d3bd567c4e8f8ca0428a8d7e869852cef445ee6da91fd/M2Crypto-0.37.1.tar.gz"
-    sha256 "e4e42f068b78ccbf113e5d0a72ae5f480f6c3ace4940b91e4fff5598cfff6fb3"
-  end
-
-  resource "pygraphviz" do
-    url "https://files.pythonhosted.org/packages/3a/d6/2c56f09ee83dbebb62c40487e4c972135661b9984fec9b30b77fb497090c/pygraphviz-1.7.zip"
-    sha256 "a7bec6609f37cf1e64898c59f075afd659106cf9356c5f387cecaa2e0cdb2304"
-  end
-
```
Comment (branchvincent) [inline Formula/dnsviz.rb:43]: seems like these are still dependencies, not sure what went wrong

Comment (chenrui333) [reply Formula/dnsviz.rb]: Good catch, it got removed when running `bump-formula-pr` command, and I cannot include  extra deps as there is some issue with `M2Crypto`

Comment (chenrui333) [reply Formula/dnsviz.rb]: This happened before as well, #69409

**Review by branchv — APPROVED** [review]

**Pull request closed without merging**